core/templates/domain/feedback_updates/feedback-updates-backend-api.service.spec.ts

```diff
@@ -0,0 +1,254 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for FeedbackUpdatesBackendApiService.
+ */
+
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+import { AddMessagePayload, FeedbackUpdatesBackendApiService } from
+  'domain/feedback_updates/feedback-updates-backend-api.service';
+
+
+describe('Feedback Updates Backend API Service', () => {
+  let feedbackUpdatesBackendApiService:
+  FeedbackUpdatesBackendApiService;
+  let httpTestingController: HttpTestingController;
+
+
+  const sampleFeedbackUpdatesDataResults = {
+    number_of_unread_threads: 0,
+    thread_summaries: [
+      {
+        status: 'open',
+        author_second_last_message: null,
+        exploration_id: 'JctD1Xvtg1eC',
+        last_message_is_read: true,
+        thread_id:
+          'exploration.JctD1Xvtg1eC.WzE1OTIyMjMzMDM3ODMuMTldWzEyOTk3XQ==',
+        author_last_message: 'nishantwrp',
+        last_updated_msecs: 1592223304062.665,
+        last_message_text: '',
+        original_author_id: 'uid_oijrdjajpkgegqmqqttxsxbbiobexugg',
+        exploration_title: 'What is a negative number?',
+        second_last_message_is_read: false,
+        total_message_count: 1
+      }
+    ]
+  };
+
+
```

Comment:
```suggestion

```
don't use two new lines, we prefer to use only 1 new line between two functions, functions & variables. here and elsewhere.

Reply:
Done, Thanks!